src/Plugin.Maui.Audio/AudioPlayer.android.cs

```diff
@@ -30,7 +30,23 @@ public double Balance
         set => SetVolume(Volume, balance = value);
     }
 
-    public bool IsPlaying => player.IsPlaying;
+	public double Speed
+	{
+		get => player.PlaybackParams.Speed;
+		set
+		{
+			try
+			{
+				player.PlaybackParams = player.PlaybackParams.SetSpeed((float)value) ?? player.PlaybackParams;
+			}
+			catch (Exception)
+			{
+				//Ensures that the speed limitations on different devices are handled!
```

Comment:
Not sure I understand this comment, why are we hiding exceptions? Should we at least consider logging it somehow so the user has some way of knowing something might be wrong?

Reply:
> Some questions, nothing shocking, I would love to see if we could keep that Android API level on 21 though

The PlaybackParams and the ability to set speed was added in API level 23. To support API level 21 we need to use the older SoundPool player. So to solve it, we should use the MediaPlayer for >= API 23 and SoundPool for < API 23. This will affect all properties and playback. But I can fix it, if API level 21 should be supported!

Reply:
> Not sure I understand this comment, why are we hiding exceptions? Should we at least consider logging it somehow so the user has some way of knowing something might be wrong?

During my tests on different Android devices, it seems like different devices has different threshold values for what speed settings that is supported. Exceeding these values result in an Exception. Not sure if there is a way to query the device for the speed threshold value!? We could add a specific exception, like a SpeedThresholdValueExceededException and throw that. Then a application could alert the user based on that exception.

Reply:
After looking more into SoundPool, it seems like it may not be a good solution for API level <23. It decompress sound into memory and has a 1 mb limitation. Great for short and multiple simultaneous sounds less than 6s. Perfect for a game engine like Orbit 😉 but useless for podcast app. Not sure there is another solution for API level 21!?